### DISCUSSION TITLE
"Administrator Initializing" prompt (SCCM) when booting from iPXE

### DISCUSSION THREAD

**Discussion by jamaya77** [ipxe/ipxe#303] [Q&A] [answered]
Hey guys,

I've modified the boot.wim file and setup a script as recommend for the sccm boot iso, but when it boots up via http, it immediately loads to a black CMD prompt that reads "Administrator Initializing" and that's where it stays. I'm using the SCCM (2010) Current Branch. What am I doing wrong?

**Modifying the boot.wim file:**
wimlib-imagex mountrw sccmfiles/sources/boot.wim 1 mntpnt
cp -drn sccmfiles/SMS/* mntpnt/sms/
rm -rf mntpnt/Windows/System32/winpeshl.ini
cp winpeshl.ini mntpnt/Windows/System32/
cp bootstrap.vbs mntpnt/sms/bin/x64/
wimlib-imagex unmount mntpnt --commit
Committing changes to /homes/amaya/Desktop/sccmfiles/sources/boot.wim (image 1)
Using LZX compression with 8 threads
Archiving file data: 40 MiB of 40 MiB (100%) done

**sccm script:**
#!ipxe

kernel wimboot
initrd bootmgr  bootmgr
initrd BCD      BCD
initrd boot.sdi boot.sdi
initrd boot.wim boot.wim
boot

<img width="828" alt="sccm_prompt" src="https://user-images.githubusercontent.com/80473349/112773771-4dd56900-9005-11eb-8c3a-8a4a22e8d3b1.png">

The unmodified boot iso I'm using, works fine via usb boot. Thanks in advance.

**Comment by oschroeder8** [comment, marked as answer]
I think this question is beyond the scope of the iPXE project. But my suggestion would be to review the content of your `winpeshl.ini`. Also, in the bottom left, there is a minimized window that might be running something requiring input.

**Reply by mcb30** [reply]
@jamaya77 Did you try following this advice?  Did it work?

Also: you may like to know that you don't have to use `wimlib-imagex` to build a modified `boot.wim` image, since `wimboot` is capable of injecting files at runtime.  See https://ipxe.org/wimboot#injected_files for details

**Comment by jamaya77** [comment]
Hi @oschroeder8 

Thank you for your time to respond. I have a follow-up question that I'm hoping is not outside the scope. I was using pxelinux before deciding to try ipxe. In that configuration, I was chain forwarding to the sccm with this configuration:

#sccm 
label sccm
      MENU LABEL ^1. Start to SCCM (x64)
      COM32 pxechn.c32
      APPEND 192.168.140.6::smsboot\x64\wdsnbp.com -W


I found a documentation where someone was using a similar configuration with ipxe, but the script fails and returns to the boot menu:
https://github.com/michaelsalisbury/builder/blob/master/ipxe/ucf.ipxe.bk04

#!ipxe

set next-server 192.168.140.6
set chain SMSBoot\x64\wdsnbp.com
autoboot 

<img width="1134" alt="forwarding" src="https://user-images.githubusercontent.com/80473349/112950337-2b784400-9108-11eb-9575-7b8f99edc4ae.png">


Can you provide any suggestions? Thanks again for your time.

**Comment by jamaya77** [comment]
@oschroeder8 
@mcb30 
Hey guys,

Thanks for following-up and for all your help. I think I'm all set with the SCCM issue. I'm not using the boot media iso, so I decided to move forward with chainloading to the SCCM instead which seems to be working great.

But for documentation purposes, I'm using SCCM Current Branch 2010 and this is my configuration with separate scripts for BIOS and UEFI  machines:


[root@ipxe sccm]# more sccmbios.ipxe 
#!ipxe

set proxydhcp/next-server 192.168.143.6
set wdsserver:ipv4 192.168.143.6
set netX/next-server ${wdsserver}
chain tftp://${wdsserver}/smsboot\x64\wdsnbp.com

[root@ipxe sccm]# more sccmuefi.ipxe 
#!ipxe

set proxydhcp/next-server 192.168.143.6
set wdsserver:ipv4 192.168.143.6
set netX/next-server ${wdsserver}
chain tftp://${wdsserver}/smsboot\x64\wdsmgfw.efi
